code/modules/mob/dead/observer/observer.dm

```diff
@@ -670,3 +670,24 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 /mob/dead/observer/proc/can_reenter_corpse()
 	var/mob/M = get_top_transmogrification()
 	return (M && M.client && can_reenter_corpse)
+
+/mob/dead/observer/AltClick(mob/user)
+	if(isAdminGhost(user))
+		var/choice_one = input(user, "Do you wish to spawn in a human?", "IC Spawning") as null|anything in list("Yes", "No")
```

Comment:
Since it's only yes/no, an alert window might be nicer for this (1 less click). Not super important though.

Reply:
I could make it a yes/no alert,